New Logo

Tobaloidee opened the pull request:
Thanks so much!

luizdepra commented:
Can you plase change the image names to use only lower case characters and `-`? It is kind of a 'good pattern' for the web.

`HUGO CODER FAVICON.svg` -> `hugo-coder-favicon.svg`

luizdepra commented on README.md:
```diff
@@ -1,3 +1,5 @@
+![hugo-coder logotype a](https://user-images.githubusercontent.com/36028424/40582328-583f656c-61a5-11e8-8d86-6e7c23f2b45d.png)
```
Please, fix this URL to something like this... following the screenshot example.
`https://github.com/luizdepra/hugo-coder/blob/master/images/hugo-coder-logotype-a.png`

 Also, you can remove the Hugo Coder title of the next line. Only showing the logo is enough.

luizdepra commented:
I agree, @khos2ow.

Tobaloidee commented:
I changed the location of the files to static/images/logo and renamed all the files to lowercases..
I had a hard time figuring out how to put the proper URL for the readme. I changed it back to the original layout.

luizdepra commented:
OK. No problem. I can change the README.md later.

Thank you very much.

khos2ow commented:
Another note, it feels to me `hugo-coder-` is redundant in the name of the files.

khos2ow commented on static/images/logo/logo:
```diff
@@ -0,0 +1 @@
+
```
What's the purpose of this empty file? Is this a mistake commit?
